pyEpiabm/pyEpiabm/intervention/travel_isolation.py

```diff
@@ -75,6 +75,15 @@ def __call__(self, time):
                                     selected_household = random.choice(
                                         existing_households)
                                     selected_household.add_person(person)
+
+                                    # Have to update the person's id to
+                                    # account for their new household
+                                    person.set_id(selected_household.id +
```

Comment:
Unfortunately without this line the test fails as it states that there are duplicate ids due to people moving household. We have discussed this with Lara, and she is going to look into it and discuss it on Wednesday

Reply:
I have changed add_household so that it accepts a new boolean parameter, which determines whether or not the id of the person is changed when add_household is called. I thought this would be a good idea as add_household is not only called in the travel isolation but is also called at different points when redistributing people between households, so it is useful to distinguish whether or not the ids of the people should be set


I wasn't sure about whether to set change_id to True or False in some places, so would be good to discuss at some point when exactly we want the id of the person to be set in all the different places